ml_service/pipelines/deploy-model-to-appservice.Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+pipeline {
+    agent { label 'master' }
+    environment {
+        ML_IMAGE_FOLDER         = 'imagefiles'
```

Comment:
Is there a way to not hard code some of these folder/file paths in the pipeline? (e.g. load from an env file or a config file like https://github.com/Merlion-Crew/MLOpsPython/blob/master/.env.example)

Reply:
We can, but this folder is used to save models in the pipeline and remove it at the  end. Therefore, I fixed it with specific folder, so we don't need unnecessary config.